conf/airframes/tudelft/bebop_opticflow_indoor_2x_30hz.xml

```diff
@@ -0,0 +1,241 @@
+<!DOCTYPE airframe SYSTEM "../airframe.dtd">
+
+<airframe name="bebop_optitrack">
+  <description>
+    Airframe file to fly a Bebop1 drone, in stable smooth flight, through waypoints using opticflow. As this airframe does not use a magnetometer nor GPS for heading determination, all positions and angles are relative to its starting position. Therefore; the drone should be facing north when started. The values are tuned so that the flight speed is limited and the drone does not over-shoot the waypoints as it would in outdoor settings.
+
+Frame size: 2 cams with 240x240
+    
+pyramid level 0 -> 30 fps average, min=29fps lowest)
+pyramid level 1 -> 26 fps average, min=15fps
+pyramid level 2 -> 21 fps average, min=11fps
```

Comment:
@dewagter this aiframe is failing with XML parsing because of the `>` characters here. You should replace the arrows with `:`.